recipes/collier/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "collier" %}
+{% set version = "1.2.8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://collier.hepforge.org/downloads/?f={{ name }}-{{ version }}.tar.gz
+  sha256: 5cb24ce24ba1f62b7a96c655b31e9fddccc603eff31e60f9033b16354a6afd89
+  folder: source
+
+build:
+  skip: true  # [win]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('collier', max_pin='x.x') }}
+  script:
+    - cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_Fortran_COMPILER=$FC -S source -B build
+    - cmake -LH build
+    - cmake --build build --clean-first --parallel="${CPU_COUNT}"
+    - cmake --install build
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}  # Not used, but needed for CMake to not error
```

Comment:
A C++ compiler isn't needed but as CMake is used, if it isn't available then the following error will occur

```
CMake Error at CMakeLists.txt:73 (project):
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.
```

The downside of this is that `collier` is now exporting a `cxx` compiler requirement into the `run` requirements automatically (c.f. https://conda-forge.org/docs/maintainer/knowledge_base/#compilers).